core/src/main/scala/com/criteo/cuttle/Database.scala

```diff
@@ -353,6 +354,15 @@ private[cuttle] case class Queries(logger: Logger) {
       (${id}, ${streams})
     """.update.run
 
+  def applyLogsRetention(logsRetention: ScalaDuration): ConnectionIO[Int] =
+    sql"""
```

Comment:
Any reason why it is not written like this ?
```sql
DELETE es
FROM executions_streams es
JOIN executions e
ON es.id = e.id
WHERE end_time < ${Instant.now.minusSeconds(logsRetention.toSeconds)}
```

Reply:
I tried but it seems not supported on Mysql. I can check again.

Reply:
It seems to work indeed.